components/centraldashboard/public/components/landing-page.js

```diff
@@ -85,40 +89,52 @@ export class LandingPage extends mixinBehaviors([AppLocalizeBehavior], utilities
                     namespaceNames.push(element.metadata.name);
                 });
                 let counter = 1;
-                const originalNs = ns;
+                // Remove any numbers at the end
+                const originalNs = ns.replace(/\d+/g, '');
                 if (namespaceNames.includes(originalNs)) {
                     while (namespaceNames.includes(originalNs + counter)) {
                         counter++;
                     }
                     ns = originalNs + counter;
                 }
                 this.namespaceName = ns;
+                this.generatedNamespace = ns;
             }).catch((e)=> {
                 this.showError('landingPage.errGeneral');
                 this.errorDetail= e;
             });
     }
 
-    async nextPage() {
-        const API = this.$.MakeNamespace;
-        API.body = {namespace: this.namespaceName};
-        this.waitForRedirect = true;
-        await API.generateRequest().completes.catch((e) => e);
-        await this.sleep(1); // So the errors and callbacks can schedule
-
-        /*
-         * Poll for profile over a span of 20 seconds (every 300ms)
-         * if still not there, let the user click next again!
-         */
-        const success = await this.pollProfile(66, 300);
-        if (success) this._successSetup();
-
-        // Create the default notebook
-        const APICreateDefault = this.$.CreateDefaultNotebook;
-
-        await APICreateDefault.generateRequest().completes.catch((e) => e);
-        await this.sleep(1); // So the errors and callbacks can schedule
+    async logout() {
+        return this.buildHref(`/logout`);
+    }
 
+    async nextPage() {
+        // Verify the email + namesoace are as expacted
+        if (this.emailAddress != this.userDetails ||
```

Comment:
I don't understand this if statement:
L58: `this.emailAddress = this.userDetails;`  -> no other sets for emailAddress or userDetails, won't this always eval to true?
Same with L101-101

Reply:
This is looking if the new data (the this.emailAddress taken from the UI, is different then the 'old' data in userDetails. If so, it means the user manually edited the code to enter what they wanted (since the inputs are disabled). Hence blocking the user.

Same logic with 101, but for the namespace